1.Exploratory_Data_Analysis/README.md

```diff
@@ -0,0 +1,7 @@
+# This folder hosts jupyter notebooks for exploratory data analysis (EDA) for multiple 'waves' of data. See this repo's main page for a link to the Interstellar preprocessing page.
+
+```Exploratory_Data_Analysis.ipynb```
+is for clustering of cells across all features. The goal is to aim to see if treatments cluster or not.
+```cell_count_analysis.ipynb``` is for qc analysis to assess if there are cell count drop offs for a given treatment.
```

Comment:
Is `qc` defined else where? If not, consider providing the full name first then use abbreviations.